packages/gatsby-theme-store/src/components/HomePage/SEO/useSitelinksSearchBoxJsonLd.ts

```diff
@@ -0,0 +1,31 @@
+import { useLocation } from '@reach/router'
+import type { PageProps } from 'gatsby'
+
+type Options = PageProps<unknown>
+
+interface Return {
+  '@context': 'https://schema.org'
+  '@type': 'WebSite'
+  url: string
+  potentialAction: {
+    '@type': 'SearchAction'
+    target: string
+    'query-input': string
+  }
+}
+
+export const useSiteLinksSearchBoxJsonLd = (_: Options): Return | null => {
```

Comment:
```suggestion
export const useSiteLinksSearchBoxJsonLd = (_: Options): Return => {
```

Reply:
I didn't really understand what you mean by this

Reply:
This function never return null right?